nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -430,6 +430,41 @@ The Label has no text when initially created. The text of the Label can be added
 choosing `Configure`
 
 
+[[component-versioning]]
+=== Component Versions
+You have access to information about the version of your Processors, Controller Services, and Reporting Tasks.
+This is especially useful when you are working within a clustered environment with multiple NiFi instances running
+different versions of a component or if you have upgraded to a newer version of a processors. The Add Processor,
+Add Controller Service, and Add Reporting Task dialogs include a column identifying the component version, as well
+as name of the component, the organization or group that created the component, and the NAR bundle that contains
+the processor. Each component displayed on the canvas also contains this information.
+
+==== Filtering Components
+When you are adding a component, you can filter based on originating source or version number.
+
+To filter based on version, click the version column to display in ascending or descending version order.
```

Comment:
The columns sort, the Source drop-down filters.  So maybe change the section to "Sorting and Filtering Components"